csrc/host_ir/executor.h

```diff
@@ -87,7 +87,8 @@ class HostIrExecutor final : public OptInDispatch {
   // Cache Fusions, FusionExecutors
   std::unordered_map<HostUnit*, FusionExecutor> fe_;
   std::unordered_map<HostUnit*, FusionExecutorCache> fec_;
-  std::unordered_map<Stream*, c10::cuda::CUDAStream> streams_;
+  std::unordered_map<Stream*, c10::cuda::CUDAStream> streams_by_pointer_;
```

Comment:
```suggestion
  std::unordered_map<Stream*, c10::cuda::CUDAStream> stream_to_cuda_stream_;
```
This makes clear to the readers what keys/values are, without having to let them find the definition.